tools/depends/target/sparkle/Makefile

```diff
@@ -0,0 +1,48 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=Sparkle
+VERSION=1.16.0
```

Comment:
can you use the latest version (1.18.1) which also includes the `sparkle:os` fix for osx

Reply:
nope - they changed something very stupid in 1.17.0 and newer. They rely on an API for logging which is only available in mac osx sdk 10.12 ... we only have osx sdk 10.11 on jenkins and i don't want to bump just for this stupid sparkle ...